targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F401xE/TARGET_STEVAL_3DP001V1/system_clock.c

```diff
@@ -154,14 +154,14 @@ uint8_t SetSysClock_PLL_HSE(uint8_t bypass)
     // Enable HSE oscillator and activate PLL with HSE as source
     RCC_OscInitStruct.OscillatorType      = RCC_OSCILLATORTYPE_HSE;
     if (bypass == 0) {
-        RCC_OscInitStruct.HSEState          = RCC_HSE_ON; // External 8 MHz xtal on OSC_IN/OSC_OUT
+        RCC_OscInitStruct.HSEState          = RCC_HSE_ON; // External 24 MHz xtal on OSC_IN/OSC_OUT
     } else {
-        RCC_OscInitStruct.HSEState          = RCC_HSE_BYPASS; // External 8 MHz clock on OSC_IN
+        RCC_OscInitStruct.HSEState          = RCC_HSE_BYPASS; // External 24 MHz clock on OSC_IN
     }
 
     RCC_OscInitStruct.PLL.PLLState = RCC_PLL_ON;
     RCC_OscInitStruct.PLL.PLLSource = RCC_PLLSOURCE_HSE;
-    RCC_OscInitStruct.PLL.PLLM            = 8;             // VCO input clock = 1 MHz (8 MHz / 8)
+    RCC_OscInitStruct.PLL.PLLM            = 24;            // VCO input clock = 1 MHz (24 MHz / 8)
```

Comment:
Clock is 25 MHz, not 24 ?

Reply:
Feel like an idiot now. Yes, it's 25Mhz. Wanting to go a bit too fast, and doing too much as once...

I could also use ```HSE_VALUE / 1000000``` now right?